### ISSUE TITLE
The dbSNP 142 file is not found.

### ISSUE THREAD

**Issue by shangyf-stu** [ikalatskaya/ISOWN#29]
Dear sir,
Thanks for your good job!
I just finish instructing the pipeline. But when I run ISOWN with the test data you provide for us, there is an error.
`annotating input file with ANNOVAR ...

The dbSNP 142 file is not found.  Please correct the path in /software/pipeline/somatic_call/ISOWN/ISOWN/bin/database_annotation.pl and try again - see path below:

	/software/pipeline/somatic_call/ISOWN/ISOWN/bin/../external_databases/dbSNP142_All_20141124.vcf.gz.modified.vcf.gz`

I am sure I have done everything followed your course, and I have tried to download dbSNP and re-format it again. 
But I still couldn't find  `dbSNP142_All_20141124.vcf.gz.modified.vcf.gz`

Any help is grateful.

Best wishes,
Shang

**Comment by shangyf-stu** [comment]
After re-format, I got three files named 
`00-All.modified.vcf.gz`, `00-All.modified.vcf.gz.tbi`, `00-All.vcf.gz`
And there is no error message.

**Comment by groadabike** [comment]
Yeah, I had the same error as @shangyf-stu. In file `database_annotation.pl` says 
```
my $dbSNP_142="$cwd/../external_databases/dbSNP142_All_20141124.vcf.gz.modified.vcf.gz";
```
I changed it to
```
my $dbSNP_142="$cwd/../external_databases/00-All.modified.vcf.gz";
```
